lib/internal/test_runner/runner.js

```diff
@@ -112,47 +121,131 @@ function getRunArgs({ path, inspectPort }) {
   return argv;
 }
 
+class FileTest extends Test {
+  #buffer = [];
+  #handleReportItem({ kind, node, level = 0, lexeme, tokens }) {
+
+    const indent = StringPrototypeRepeat(kDefaultIndent, level + 1);
+
+    const details = (diagnostic) => {
+      return (
+        diagnostic && {
+          __proto__: null,
+          yaml:
+            `${indent}  ` +
+            ArrayPrototypeJoin(diagnostic, `\n${indent}  `) +
+            '\n',
+        }
+      );
+    };
+
+    switch (kind) {
+      case TokenKind.TAP_VERSION:
+        break; // skip
+
+      case TokenKind.TAP_PLAN:
+        this.reporter.plan(indent, node.plan.end - node.plan.start + 1);
+        break;
+
+      case TokenKind.TAP_SUBTEST_POINT:
+        this.reporter.subtest(indent, node.name);
+        break;
+
+      case TokenKind.TAP_TEST_OK:
+      case TokenKind.TAP_TEST_NOTOK:
+        const { todo, skip, pass, fail } = node.test.status;
+        const testName = node.test.description;
+        const testnumber = node.test.id;
+        let directive = {};
+
+        if (todo) {
+          this.skip(node.test.reason);
+          directive = this.getTodo(node.test.reason);
+        }
+
+        if (skip) {
+          this.todo(node.test.reason);
```

Comment:
same question? it is only the child test that has a todo dirctive

Reply:
I will remove these calls. I noticed the details during the postrun were miscalculated. I was figuring out why. Any idea what causes this?

![image](https://user-images.githubusercontent.com/1699357/191917235-8a17432b-3dd1-4fc7-8f49-4405193fb4fc.png)

__